### PR TITLE
chore: bump up Go to 1.15

### PR DESCRIPTION
**Pull request by knqyf263**
darwin/386 is no longer supported.
https://golang.org/doc/go1.15#darwin

### CODE REVIEW

File: goreleaser.yml [hunk 1]
```diff
@@ -19,6 +19,9 @@ builds:
       - arm64
     goarm:
       - 7
+    ignore:
```
Comment (simar7) [inline goreleaser.yml:22]: I can't leave a comment on that line but should this be removed? https://github.com/aquasecurity/trivy/pull/646/files#diff-39e929907e3737ff7460d982cb6fbc60R41

Comment (simar7) [reply goreleaser.yml:22]: And this https://github.com/aquasecurity/trivy/pull/646/files#diff-39e929907e3737ff7460d982cb6fbc60R17

Or I guess in general all occurrences of 386?

Comment (knqyf263) [reply goreleaser.yml:22]: We should continue to support linux/386. This `ignore` section affects only darwin/386.

Comment (simar7) [reply goreleaser.yml:22]: Oh I see – I thought it was 386 in general.

**Review by simar7 — APPROVED** [review]

**Pull request merged**